jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -12,31 +61,42 @@ def step000_provision_sut(SKIP_PROVISIONING, script_dir) {
     }
 }
 
-def step010_setup_beaker(script_dir) {
-    withEnv(["SUT_HOST=${SUT_HOST}"]) {
+def step010_setup_beaker(script_dir, server_version) {
+    if (server_version["type"] != "pe") {
+        error "Unsupported server type: ${server_version["type"]}"
+    }
+
+    withEnv(["SUT_HOST=${SUT_HOST}",
+             "pe_version=${server_version["pe_version"]}",
```

Comment:
I believe that is correct but I was planning on digging into the beaker docs later to sort that out.  I'm fine with just released versions for this pass.